items/items_init.py

```diff
@@ -0,0 +1,28 @@
+from items.weapon import Weapon
+from items.armour import Armour
+from items.potion import Potion
+
+
+class InitItems:

```

Comment:
Not really descriptive name for a class. You can add all this methods to the Weapon/Armour/Potion classes, it will look much more better and understandable. Don't forget about OOP